generator/typefile.go

```diff
@@ -81,6 +81,7 @@ func (file *typeFile) parseAttributes(namePrefix string, typePrefix string, attr
 	fields := make([]structField, 0, len(attr))
 
 	for _, name := range sortKeys(attr) {
+		//fmt.Println("parsing: ", namePrefix, typePrefix, name)
```

Comment:
Do not leave commented lines in the code base

Reply:
thanks, removed